src/app/AttributeAccessInterface.h

```diff
@@ -352,12 +352,18 @@ class AttributeAccessInterface
      *
      * @param [in] aPath indicates which exact data is being read.
      * @param [in] aEncoder the AttributeValueEncoder to use for encoding the
-     *             data.  If this function returns scucess and no attempt is
-     *             made to encode data using aEncoder, the
-     *             AttributeAccessInterface did not try to provide any data.  In
-     *             this case, normal attribute access will happen for the read.
-     *             This may involve reading from the attribute store or external
-     *             attribute callbacks.
+     *             data.
+     *
+     * The implementation can do one of three things:
+     *
+     * 1) Return a failure.  This is treated as a failed read and the error is
+     *    returned to the client, by converting it to a StatusIB.
```

Comment:
This is actually currently dropped: everthing unwinds without a proper error it seems.

Reply:
Yep, see the NOTE above.  I am working on a fix for that.

Reply:
https://github.com/project-chip/connectedhomeip/pull/13950